auth-web/src/components/auth/staff/SetupAccountForm.vue

```diff
@@ -227,6 +227,7 @@ export default class SetupAccountForm extends Vue {
         await this.createInvitation({
           recipientEmail: this.email,
           sentDate: new Date(),
+          type: 'DIRECTOR_SEARCH',
```

Comment:
Type should be a generic one? just that it should not be DIRECTOR_SEARCH for all the products, right?